controllers/admin/AdminAdyenPrestashopCronController.php

```diff
@@ -13,13 +13,15 @@
  *                               #############
  *                               ############
  *
- * Adyen PrestaShop module
+ * Adyen PrestaShop plugin
```

Comment:
Prestashop uses the temonology modules https://addons.prestashop.com/en/ shall we call it a module ? ask Grant to verify this.